velox/exec/benchmarks/HashJoinListResultBenchmark.cpp

```diff
@@ -462,6 +462,7 @@ class HashTableListJoinResultBenchmark : public VectorTestBase {
     eraseTime_ += eraseClock.timeToDropValue();
   }
 
+  std::default_random_engine randomEngine;
```

Comment:
Initialize this with `random_device` in the constructor, otherwise it will give same values all the time